### ISSUE TITLE
Suggested new risk "Sustainable funding"

### ISSUE THREAD

**Issue by kbryk96** [CWWhitney/nifam_codas_school_garden#59]
The success and sustainability of a school garden is very dependent on the ongoing support of the government, policy, funding

**Comment by CWWhitney** [comment]
Shall we add this... 
Add an 'ongoing support of the government, policy, funding' risk to the long-term benefit?

**Comment by kbryk96** [comment]
@CWWhitney yes I think that it definitely should be included! "finding sustained sources of funding for school-based intervention... can pose a challenge for many low- and middle-income countries" ([2](https://github.com/CWWhitney/nifam_codas_school_garden/wiki/x-References)).

Ongoing funding is one of the most commonly cited challenges ([Yu, 2012](https://udspace.udel.edu/items/5bdc9007-8b94-479d-8108-2f0f8a736f7a))
